src/index.html

```diff
@@ -5,9 +5,244 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="icon" href="./images/favicon.png">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <div class="header__background">
+        <div class="header__background-image-red-filter">
+          <div class="header__nav">
+            <nav class="nav">
```

Comment:
```suggestion
          <nav class="header__nav nav">
```
Why not to use mix of classes instead of this div?